src/main/java/io/github/oliviercailloux/y2018/apartments/valuefunction/ValueDistFunction.java

```diff
@@ -47,8 +47,7 @@ public void addInterestLocation(LatLng interest) throws Exception {
 			maxDuration = currentdistance;
```

Comment:
à la ligne 30, serait-il possible de documenter le deuxième paramètre stp ?

Reply:
Fixed in commit ea9579e